CMakeLists.txt

```diff
@@ -220,6 +221,15 @@ else()
   set(PARANEURON 0)
 endif()
 
+# =============================================================================
+# Enable backward
+# =============================================================================
+if(NRN_ENABLE_BACKTRACE)
+    set(USE_BACKWARD 1)
```

Comment:
just to not conflict with other submodule projects, should set `NRN_USE_BACKWARD` ?